### ISSUE TITLE
804a messages are not processed

### ISSUE THREAD

**Issue by pipiche38** [sasu-drooz/Domoticz-Zigate#189]
I'm receiving time to time such message (when activating debug mode )

```
Sep 17 08:48:36 pi3 domoticz[921]: 2018-09-17 08:48:36.069  (Zigate) ZigateRead - MsgType 804a - Reception Management Network Update response : 01804a001ca900000023000907fff80010b8c4c1b0b5a8aba9aaa8bfced2bda4aa4503
Sep 17 09:04:23 pi3 domoticz[921]: 2018-09-17 09:04:23.564  (Zigate) ZigateRead - MsgType 804a - Reception Management Network Update response : 01804a001ce300000016000707fff80010b3c2beacb4a6aca9aaa7c0acd2bea9a14503
```

**Comment by zaraki673** [comment]
see 6fbe7f4

**Comment by zaraki673** [comment]
Channel list has to be read

**Comment by pipiche38** [comment]
Implemented in beta-3.2
